### PR TITLE
fix: [TokenDetails] Glyph placement on line chart

### PR DESCRIPTION
**Pull request by gbugyis**
- Uses d3 curveCardinalOpen instead of d3-curve-circlecorners to fix inconsistency between glyph placement and chart line on steep data points.

### CODE REVIEW

File: src/components/Charts/LineChart.tsx [hunk 1]
```diff
@@ -37,7 +36,7 @@ function LineChart<T>({
     <svg width={width} height={height}>
       <Group top={marginTop}>
         <LinePath
-          curve={curve ?? radius(0.25)}
+          curve={curve}
```
Comment (cartcrom) [inline src/components/Charts/LineChart.tsx:39]: curve should potentially be a required prop here now, or we can choose a new default

File: src/components/Tokens/TokenDetails/PriceChart.tsx [hunk 2]
```diff
@@ -246,8 +249,7 @@ export function PriceChart({ width, height, token }: PriceChartProps) {
         getX={(p: PricePoint) => timeScale(p.timestamp)}
         getY={(p: PricePoint) => rdScale(p.value)}
         marginTop={margin.top}
-        /* Default curve doesn't look good for the ALL chart */
-        curve={timePeriod === TimePeriod.ALL ? curveBasis : undefined}
+        curve={curveCardinalOpen.tension(curveTension)}
```
Comment (cartcrom) [inline src/components/Tokens/TokenDetails/PriceChart.tsx:252]: awesome

File: src/components/Charts/LineChart.tsx (hunk 1 above)
Comment (gbugyis) [reply src/components/Charts/LineChart.tsx:39]: ah, good catch. I made it required now

File: package.json [hunk 3]
```diff
@@ -175,7 +175,6 @@
     "clsx": "^1.1.1",
     "copy-to-clipboard": "^3.2.0",
     "d3": "^7.6.1",
-    "d3-curve-circlecorners": "^0.1.6",
```
Comment (cartcrom) [inline package.json:178]: One last thing, could you remove the d3-curve-circlecorners from react-app-env.d.ts

**Review by cartcrom — CHANGES_REQUESTED** [review]

**Review by cartcrom — APPROVED** [review]

**Pull request merged**